Signal/src/util/Pastelog.m

```diff
@@ -58,6 +58,14 @@ +(void)submitLogs {
                                          handler:^(UIAlertAction *_Nonnull action) {
                                              [Pastelog.sharedManager sendToSelf:urlString];
                                          }]];
+            [alert addAction:[UIAlertAction
+                                 actionWithTitle:
+                                     NSLocalizedString(@"DEBUG_LOG_ALERT_OPTION_SEND_TO_LAST_THREAD",
+                                         @"Label for the 'send to last thread' option of the the debug log alert.")
+                                           style:UIAlertActionStyleDefault
+                                         handler:^(UIAlertAction *_Nonnull action) {
+                                             [Pastelog.sharedManager sendToLastThread:urlString];
```

Comment:
In DEBUG builds, I usually want to send the debug log to myself via a group which is usually the first thread.